doc/Config.md

```diff
@@ -37,8 +37,8 @@ The `config` directory holds key configuration files and can have different conf
 All of them have to end with `.config.php` and must not include `-sample` in their name.
 
 Some examples of common known configuration files:
-- `local.config.php` holds the current node custom configuration.
-- `addon.config.php` is optional and holds the custom configuration for specific addons.
+- `local.config.php` holds the base node custom configuration.
+- Any other file in this folder is meant for addon configuration.
```

Comment:
Any other file is loaded as well in no order ..
have a look at all the configs at https://github.com/friendica/docker/tree/stable/.config
That was one of the main goals for the `Config` changes in the past :)

Reply:
No order?

Reply:
ok sry, too much DEnglisch *gg*

The only order inside `config/` is the "standard" directory order, means that `local.config.php` isn't even necessary anymore. Any config file will be loaded, regardless it's naming (except the postfix `config.php`)

Reply:
It's true, but the `local.config.php` file is tied to the `local-sample.config.php` we provide, and it's also the name of the file we create during automatic install. So it isn't technically necessary, but it still serves a couple purposes.

Reply:
```suggestion
- Any other file in this folder is meant for additional configuration (e.g. for addons).
```